dhd.csl

```diff
@@ -0,0 +1,186 @@
+      <text variable="collection-number"/>
+    </group>
+  </macro>
+  <macro name="page-range">
+    <group delimiter=" " suffix=".">
+      <text variable="page"/>
+    </group>
+  </macro>
+  <macro name="online-url">
+    <group delimiter=" ">
+      <text variable="URL"/>
+      <date form="text" variable="accessed" prefix="[letzter Zugriff " suffix="]"/>
+    </group>
+  </macro>
+  <macro name="container-title-volume">
+    <group delimiter=" " suffix=":">
+      <text variable="container-title" font-style="italic"/>
+      <text variable="volume"/>
+      <text variable="issue" prefix="(" suffix=")"/>
+    </group>
+  </macro>
+  <macro name="creator-short">
+    <names variable="author">
+      <name form="short" delimiter=" / "/>
+      <substitute>
+        <names variable="editor"/>
+      </substitute>
+    </names>
+  </macro>
+  <citation et-al-min="4" et-al-use-first="1" disambiguate-add-year-suffix="true" collapse="year-suffix">
+    <layout delimiter="; " prefix="(" suffix=")">
+      <group delimiter=": ">
+        <text macro="creator-year-short"/>
+        <group delimiter=" ">
+          <text variable="locator"/>
+        </group>
+      </group>
+    </layout>
+  </citation>
+  <bibliography et-al-min="9" et-al-use-first="1" entry-spacing="0" hanging-indent="true">
+    <sort>
+      <key macro="creator-short"/>
+      <key macro="creator-year-short"/>
+    </sort>
+    <layout>
+      <group delimiter=" " suffix=".">
+        <group delimiter=" ">
+          <text macro="creator-year-long" suffix=":"/>
+        </group>
+        <choose>
+          <if type="book" match="any">
+            <group delimiter=" ">
+              <text variable="title" font-style="italic" suffix="."/>
+              <text macro="collection-title-number"/>
+              <text macro="edition"/>
+              <text macro="publication-place-publisher"/>
+            </group>
+          </if>
+          <else-if type="article-journal article-magazine article-newspaper" match="any">
+            <text variable="title" prefix="&quot;" suffix="&quot;, "/>
+            <group delimiter=" ">
+              <group delimiter=": ">
+                <text term="in"/>
+                <text macro="container-title-volume"/>
+              </group>
+              <text macro="page-range"/>
+              <text variable="DOI" strip-periods="false"/>
+            </group>
+          </else-if>
+          <else-if type="chapter" match="any">
+            <group delimiter=" ">
+              <text variable="title" prefix="&quot;" suffix="&quot;"/>
+              <text macro="editor-container-title"/>
+              <text macro="collection-title-number"/>
+              <text macro="edition"/>
+              <text macro="publication-place-publisher"/>
+              <text macro="page-range"/>
+              <text variable="DOI"/>
+            </group>
+          </else-if>
+          <else-if type="dataset" match="any"/>
```

Comment:
Is it intended that datasets are omitted from the bibliography?

Reply:
I would like to leave this open to the Germanophone DH community who will meet in two weeks to discuss the requirements